### ISSUE TITLE
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.

### ISSUE THREAD

**Issue by msftedad** [uxsolutions/bootstrap-datepicker#2719]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7653)
Browser: Edge (Version 144.0.3719.82)
URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

**User Impact:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), users with low vision or those relying on keyboard navigation cannot identify the selected date or focus indicator on calendar controls because they are not visible. This makes scheduling and date selection difficult, reducing accessibility and usability.

**Pre-Requisites:**
Setting > Accessibility > Contrast Themes > (Aquatic/Desert/Dusk/Night Sky)

**Repro Steps:**

1. Open the URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ in edge browser in windows.
2. Press tab key to navigate to the 'Select Date' control.
3. Calendar will open. 
4. Press tab key to navigate to the calendar dates. 
5. Verify whether in high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are visible in windows or not.

**Actual Result:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.

**Refer Attachment:**

1. In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.png
2. In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.mp4

https://github.com/user-attachments/assets/a2060946-7ef4-457a-bfda-9e897490c1bf
<img width="2945" height="955" alt="Image" src="https://github.com/user-attachments/assets/743e3625-0060-46c0-99db-c2f341dc3e06" />

**Expected Result:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls should be visible in windows.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
